README.md

```diff
@@ -25,85 +24,139 @@ Table of Contents:
   - Monitor the Routing Engine status
 -->
 
-<br>
-
 ## Goal of this repository
 
-**Learn thourgh practical examples how to create a custom IBA probe, from the definition of a Custom Telemetry Collector to using it in a Custom IBA probe.**
+**Learn through practical examples how to create a custom IBA probe, from the definition of a Custom Telemetry Collector to using it in a Custom IBA probe.**
 
-Creating new IBA probe involves several parts of Apstra (Telemetry, GraphDB, IBA). Each part has a role to play, the examples are selected and described with the ambition to show these different parts and how they come together. Going through the examples you will notice some of them have more complexity at the collector level because we need to process the data in a certain way to make it effectively usable by the analytics capabilities of IBA. Other times you will find simple collectors but more sophisticated IBA probes because the use case requires the implementation of an advanced processing of this data in order to extract the desired insight.
+Creating new IBA probe involves several parts of Apstra (Telemetry, GraphDB, IBA). Each part has a role to play, and the examples are selected and described with the ambition to show these different parts and how they come together. Going through the examples, you will notice some of them have more complexity at the collector level because we need to process the data in a certain way to make it effectively usable by the analytics capabilities of IBA. Other times you will find simple collectors but more sophisticated IBA probes because the use case requires the implementation of an advanced processing of this data to extract the desired insight.
 
 > [!IMPORTANT]
 > The examples in this repository are provided for educational purposes and are expected to be tested and customised to your specific needs before deploying them in your production blueprints.
 
-<br>
-
 ## Content of this repository
-The following examples of IBA probes are listed in this repository.
+Here are some examples of IBA probes we have looked at since `4.2.1`.
 
-1) [Ping Mesh](Ping_Mesh/release_4.2.1/README.md) 
-2) [Device Uptime](Device_Uptime/release_4.2.1/README.md) 
-3) [System Alarm](System_Alarm/release_4.2.1/README.md)
-4) [OSPF Neighbor](OSPF_Neighbor/release_4.2.1/README.md)
-5) [Interface Flap](Interface_Flap/release_4.2.1/README.md)
-6) [OS Version Compliance](OS_Version_Compliance/release_4.2.1/README.md)
-7) [DDoS Protection Protocols](DDoS_Protection_Protocols/release_4.2.1/README.md)
-8) [Interface Queue](Interface_Queue/release_4.2.1/README.md) <<< WIP
-9) [Lightweight Loop Detection](Loop_Detection/release_4.2.1/README.md)
+1) Ping Mesh
+2) Device Uptime
+3) System Alarm
+4) OSPF Neighbor
+5) Interface Flap
+6) OS Version Compliance
+7) DDoS Protection Protocols
+8) Interface Queue
```

Comment:
We can remove this, we are delivering a built-in probe for Interface Queue stats as part of the sprints and it will leverage the creation of a new source processor, since we are not able to acheive what we want just with custom collectors in this case. Suggest removing this.